packages/next/pages/_document.tsx

```diff
@@ -89,6 +137,13 @@ function getPreNextScripts(context: HtmlProps, props: OriginProps) {
       )
     }
   )
+
+  return (
+    <>
+      {webWorkerScripts}
```

Comment:
Adding note. The inline snippets should be ideally placed before CSS files. I have another PR for this and we can merge the logic